obspy/io/nordic/core.py

```diff
@@ -50,51 +57,74 @@ def __init__(self, value):
         self.value = value
 
 
-def _is_sfile(sfile):
+def _is_sfile(sfile, encoding='latin-1'):
     """
     Basic test of whether the file is nordic format or not.
 
     Not exhaustive, but checks some of the basics.
 
     :type sfile: str
     :param sfile: Path to sfile
+    :type encoding: str
+    :param encoding: Encoding for file, used to decode from bytes to string
+
     :rtype: bool
     """
     if not hasattr(sfile, "readline"):
         try:
-            with open(sfile, 'r') as f:
-                head_line = _get_headline(f)
+            with open(sfile, 'r', encoding=encoding) as f:
+                tags = _get_line_tags(f=f)
         except Exception:
             return False
     else:
-        head_line = _get_headline(sfile)
-    if head_line is not None:
+        tags = _get_line_tags(f=sfile)
+    if tags is not None:
+        # Note that there can be two origin lines, but only to allow more
+        # magnitudes, not more origins.
+        try:
+            head_line = tags['1'][0][0]
+        except KeyError:
+            return False
         try:
             sfile_seconds = int(head_line[16:18])
         except ValueError:
             return False
         if sfile_seconds == 60:
             sfile_seconds = 0
         try:
-            UTCDateTime(int(head_line[1:5]), int(head_line[6:8]),
-                        int(head_line[8:10]), int(head_line[11:13]),
-                        int(head_line[13:15]), sfile_seconds,
-                        int(head_line[19:20]) * 100000)
+            UTCDateTime(
+                int(head_line[1:5]), int(head_line[6:8]), int(head_line[8:10]),
+                int(head_line[11:13]), int(head_line[13:15]), sfile_seconds,
+                int(head_line[19:20]) * 100000)
             return True
         except Exception:
             return False
     else:
         return False
 
 
-def _get_headline(f):
+def _get_line_tags(f):
+    f.seek(0)
+    line = f.readline()
+    if len(line.rstrip('\n').rstrip('\r')) != 80:
```

Comment:
Just `.rstrip('\n\r')` should be enough. Stripping always removes as many of those. Just using `.rstrip()` should actually also be enough as it by default will remove any ASCII whitespace I believe.